app/uk/gov/hmrc/nisp/views/excluded_sp_old.scala.html

```diff
@@ -37,10 +37,14 @@
     @main(browserTitle=Some(Messages("nisp.main.title")), userLoggedIn = true,
         pageTitle = Some(Messages("nisp.excluded.title")), analyticsAdditionalJs = Some(analyticsAdditionalJs)) {
 
+        @if(spExclusions.exclusions.contains(Exclusion.ManualCorrespondenceIndicator)) {
+            @exclusions.dead()
+            ???
```

Comment:
Again I don't think we need an MCI if statement, and it shouldn't go the dead exclusion or produce ???